pkg/apis/admissionregistration/types.go

```diff
@@ -247,6 +247,11 @@ type ValidatingAdmissionPolicySpec struct {
 	// A maximum of 20 auditAnnotation are allowed per ValidatingAdmissionPolicy.
 	// +optional
 	AuditAnnotations []AuditAnnotation
+
+	// Variables contain definitions of variables that can be used in composition of other expressions.
+	// The variables defined here will be available under `variables` in other expressions.
```

Comment:
If we're allowing variables to access variables defined earlier in the list, say that here too?

Also, explicitly say that each variable is a CEL expression? It's obvious to us, but users might not realize that immediately.

Reply:
Good point. updated.